### PR TITLE
cleared out html files and updates README

### PR DESCRIPTION
**Pull request by aiden-mcmanus**
Changed the location of html files that will be used in future years to be in "currentyear" instead of 2023. Removed bloat files that are not being used anymore such as compselect.html or qr code files. Put recurring css content into one file. Updated readme to include Apple users. Removed mentions of 2023 and pit related content. Added a funny lil secret for max :)

### CODE REVIEW

File: names.txt
Comment (LuigisLuigi) [inline names.txt:1]: ??????????? what are those names???

File: static/2023/js/qrcode.js
Comment (LuigisLuigi) [inline static/2023/js/qrcode.js:1]: 😢

File: static/css/commons.css
Comment (LuigisLuigi) [inline static/css/commons.css:1]: very good, i love this ⭐

File: static/currentyear/js/scout.js
Comment (LuigisLuigi) [inline static/currentyear/js/scout.js:1]: change the cones and cubes to something else, just not that because it'll be different every year

**Review by LuigisLuigi — CHANGES_REQUESTED** [review]
change the cubes and cones in scout.js to literally anything else, not anything, you know what I mean. after that, it all looks great and I'll approve it

File: names.txt
Comment (aiden-mcmanus) [reply names.txt:1]: one piece

**Review by LuigisLuigi — APPROVED** [review]

**Pull request merged**